go/store/nbs/archive_build.go

```diff
@@ -346,6 +406,9 @@ func verifyAllChunks(idx tableIndex, archiveFile string) error {
 			msg := fmt.Sprintf("hash mismatch for chunk: %s", h.String())
 			return errors.New(msg)
 		}
+
+		chunkProgress++
+		progress <- ArchiveBuildProgressMsg{Stage: "Verifying Chunks", Total: chunkCount, Completed: chunkProgress}
```

Comment:
(Minor/pedantic) The progress stages I saw are named: 
* Chunk Groups Materialized
* Ungrouped Chunks Written
* Building Chunk Group Dictionaries
* Verifying Chunks

Totally minor and maybe a bit pedantic, but might be nice to name them with a consistent form, such as "Materializing Chunk Groups", "Writing Ungrouped Chunks", "Building ...", "Verifying ...". Then they all start with a gerund and imply that stage is in actively happening.